app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :deployments_node_deployment
```

Comment:
Your right, fixing.
I don't this so, every vm can serve as a node/master in one deployment of OpenShift.